packages/blade/src/components/Radio/RadioIcon/RadioIconWrapperStyles.ts

```diff
@@ -72,7 +72,7 @@ const getRadioIconWrapperStyles = ({
     marginRight: makeSpace(theme.spacing[2]),
     backgroundColor: getIn(theme, backgroundColor),
     borderColor: getIn(theme, borderColor),
-    transitionDuration: makeMotionTime(theme.motion.duration.xquick),
+    transitionDuration: (makeMotionTime(theme.motion.duration.xquick) as unknown) as string,
```

Comment:
Now that we have proper type support shouldn't we be getting rid of such hacks?

Reply:
We have proper types support for consumers, from blade side we have to ensure that our components are compatible with both platform types. so here makeMotion time returns different type on different platform but the file is isomorphic (without any extension).